src/Security/Security.php

```diff
@@ -1266,6 +1266,14 @@ public static function force_database_is_ready($isReady)
      */
     private static $default_login_dest = "";
 
+    /**
+     * @config
+     * @var string Set the default reset password destination
+     * This is the URL that users will be redirected to after they change their password,
+     * By default, it's redirecting to {@link $login}.
+     */
+    private static $default_reset_password_dest = false;
```

Comment:
Set to `false`, so the actual handler will fall back to the login link instead of breaking when it's an empty string (happens on strict type-checks)

Reply:
Have you verified this?

Reply:
It happened on my machine earlier, can't reproduce this anymore, so `null` is the way to go then :)